src/clib/lib/enkf/ensemble_config.cpp

```diff
@@ -88,25 +88,28 @@ void ensemble_config_add_node(ensemble_config_type *ensemble_config,
             "** Warning: Invalid node passed to ensemble_config_add_node\n");
 }
 
-void ensemble_config_init_SUMMARY_full(ensemble_config_type *ensemble_config,
-                                       const char *key,
-                                       const ecl_sum_type *refcase) {
-    ensemble_config->summary_keys.push_back(std::string(key));
+std::list<std::string>
+ensemble_config_get_summary_key_list(ensemble_config_type *ensemble_config,
+                                     const char *key,
+                                     const ecl_sum_type *refcase) {
+    std::list<std::string> keylist;
     if (util_string_has_wildcard(key)) {
         if (refcase != NULL) {
             stringlist_type *keys = stringlist_alloc_new();
             ecl_sum_select_matching_general_var_list(
                 refcase, key,
                 keys); /* expanding the wildcard notation with help of the refcase. */
             int k;
```

Comment:
Could probably move to `for(int k=0...` while we are here.

Reply:
Yep, also removed unused argument. 👍